junit5-maven-consumer/pom.xml

```diff
@@ -43,6 +44,21 @@
 						<artifactId>junit-platform-surefire-provider</artifactId>
 						<version>${junit.platform.version}</version>
 					</dependency>
+					<dependency>
+						<groupId>org.junit.jupiter</groupId>
+						<artifactId>junit-jupiter-engine</artifactId>
+						<version>${junit.jupiter.version}</version>
+					</dependency>
+					<dependency>
+						<groupId>org.junit.vintage</groupId>
+						<artifactId>junit-vintage-engine</artifactId>
+						<version>${junit.vintage.version}</version>
+					</dependency>
+					<dependency>
+						<groupId>org.junit.platform</groupId>
+						<artifactId>junit-platform-runner</artifactId>
+						<version>${junit.platform.version}</version>
```

Comment:
Why `junit-platform-runner`? If that were used it would need to be a `test` dependency, because it would be required by `test-compile`, right?